src/components/Layout.jsx

```diff
@@ -1,6 +1,6 @@
-import '@reshuffle/code-transform/macro';
-import React from 'react';
-import { useAuth } from '@reshuffle/react-auth';
+import "@reshuffle/code-transform/macro";
+import React from "react";
+import { useAuth } from "@reshuffle/react-auth";
```

Comment:
it was changed from a single quote to double, you can set your project/IDE to fix it back.

Reply:
Changed it thank you for the heads up!